packages/block-library/src/audio/test/index.js

```diff
@@ -1,13 +0,0 @@
-/**
- * Internal dependencies
- */
-import { name, settings } from '../';
```

Comment:
Seems a shame to remove these tests. I spent a while trying to find a way to mock 'select' and the hasUploadPermissions selector, but couldn't find a suitable way. Any pointers would be appreciated if there is a way to do this.

But I also thought that perhaps it would be better to spend the effort introducing e2e tests that cover the initial state of these blocks?